Data/Context/EcommerceContext.cs

```diff
@@ -0,0 +1,101 @@
+﻿using EFCoreConsoleApp.Data.Entities;
+using Microsoft.EntityFrameworkCore;
+
+namespace EFCoreConsoleApp.Data.Context
+{
+    public class EcommerceContext : DbContext
+    {
+        public EcommerceContext()
+        {
+        }
+
+        protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
+        {
+            optionsBuilder.UseNpgsql("Server=127.0.0.1;Port=5432;Database=EFCoreConsole;User Id=postgres;Password=Sancho38293!;");
```

Comment:
Have you tried to apply migrations to your local db? Seems like it it will not work, as at least you have "User Id" param defined with whitespace. And also change db name to EcommerceStore or something meaningful
Please fix it and apply the migrations to the local db